cmd/buildah/mount.go

```diff
@@ -42,17 +42,21 @@ func mountCmd(c *cli.Context) error {
 	}
 	truncate := !c.Bool("notruncate")
 
-	if len(args) == 1 {
-		name := args[0]
-		builder, err := openBuilder(getContext(), store, name)
-		if err != nil {
-			return errors.Wrapf(err, "error reading build container %q", name)
-		}
-		mountPoint, err := builder.Mount(builder.MountLabel)
-		if err != nil {
-			return errors.Wrapf(err, "error mounting %q container %q", name, builder.Container)
+	var lastError error
+	if len(args) > 0 {
+		for _, name := range args {
+			builder, err := openBuilder(getContext(), store, name)
+			if err != nil {
+				lastError = util.WriteError(os.Stderr, errors.Wrapf(err, "error reading build container %q", name), lastError)
+				continue
+			}
+			mountPoint, err := builder.Mount(builder.MountLabel)
+			if err != nil {
+				lastError = util.WriteError(os.Stderr, errors.Wrapf(err, "error mounting %q container %q", name, builder.Container), lastError)
+				continue
+			}
+			fmt.Printf("%s\n", mountPoint)
```

Comment:
When I ran with this command:
```
# buildah mount $cid1 bad $cid2 bad3 $cid3
/var/lib/containers/storage/overlay/2ce6844f2f162dd5f8af6711b063f68cec9ab07cd414713a837b3c291b0b82a4/merged
/var/lib/containers/storage/overlay/5554fdd9f5012887c74fc2fb17cdd3854db933b1c1f6a1f55e9ea5bb99a4602d/merged
error reading build container "bad": error reading build container: container not known
/var/lib/containers/storage/overlay/63638ee3a784c3deb5318c848749165f7d40f491a3ebe000f4d61cdbd157bb1f/merged
error reading build container "bad3": error reading build container: container not known
```
The user could map the container to the mount with just 'buildah mount', but I wonder if it would make sense to print out the container name before the mount if there is more than one mount requested?  If it's a single I would not add the container name to make scripting easier.   What I'm thinking is changing line 58 to:

```
                        if len(args) >1 {
                                fmt.Printf("%s %s\n", name, mountPoint)
                        } else {
                                fmt.Printf("%s\n", mountPoint)
                        }

```
which ends up showing:

```
buildah mount $cid1 bad $cid2 bad3 $cid3
alpine-working-container /var/lib/containers/storage/overlay/2ce6844f2f162dd5f8af6711b063f68cec9ab07cd414713a837b3c291b0b82a4/merged
alpine-working-container-2 /var/lib/containers/storage/overlay/5554fdd9f5012887c74fc2fb17cdd3854db933b1c1f6a1f55e9ea5bb99a4602d/merged
error reading build container "bad": error reading build container: container not known
alpine-working-container-3 /var/lib/containers/storage/overlay/63638ee3a784c3deb5318c848749165f7d40f491a3ebe000f4d61cdbd157bb1f/merged
error reading build container "bad3": error reading build container: container not known
```
Thoughts?

Reply:
@TomSweeneyRedHat I really like this idea 👍 

When I saw this PR I wasn't sure how useful it would be given how I generally consume the output for use elsewhere in scripts. This would make the output much cleaner for consumption.

Reply:
@q384566678 this seems like a good idea, could you make this change and then document it in the man page.